apps/ai-image-generator/backend/package.json

```diff
@@ -3,18 +3,29 @@
   "version": "0.1.0",
   "private": true,
   "description": "",
-  "type": "module",
   "main": "index.js",
   "scripts": {
-    "validate-actions": "node scripts/validate-actions.js",
-    "build-actions": "NODE_ENV=production tsc",
-    "build": "npm run validate-actions && npm run build-actions"
+    "validate-actions": "node scripts/validate-actions.mjs",
+    "build-actions": "NODE_ENV=production tsc --module esnext",
```

Comment:
Would this work with `commonjs`? Asking because I saw this https://stackoverflow.com/questions/60993812/mocha-typescript-cannot-use-import-statement-outside-a-module when looking at your above comment about `mocha` and `typescript`

Reply:
No I'm afraid not. The problem is the commonjs format is what's not compatible in our lambda runtime. 🤷‍♂️